src/modules/song/slice.ts

```diff
@@ -1,238 +1,238 @@
+    //   contributors: {
+    //     1: { name: "John", role: Role.Producer, stake: 0.25 },
+    //     2: { name: "Dan", role: Role.SoundEngineer, stake: 0.25 },
+    //     3: { name: "Kid Cudi", role: Role.Singer, stake: 0.5 },
+    //   },
+    //   extraInformation: "extra info",
+    // },
+    // 11: {
+    //   name: "Sad people",
+    //   id: 5,
+    //   genre: "Hip Hop",
+    //   userRole: Role.Singer,
+    //   releaseDate: new Date(2018, 5).toString(),
+    //   description: "Description",
+    //   albumImage:
+    //     "https://images.complex.com/complex/images/c_fill,dpr_auto,f_auto,q_90,w_1400/fl_lossy,pg_1/hcjrqlvc6dfhpjxob9nt/cudi",
+    //   contributors: {
+    //     1: { name: "John", role: Role.Producer, stake: 0.25 },
+    //     2: { name: "Dan", role: Role.SoundEngineer, stake: 0.25 },
+    //     3: { name: "Kid Cudi", role: Role.Singer, stake: 0.5 },
+    //   },
+    //   extraInformation: "extra info",
+    // },
+    // 12: {
+    //   name: "Sad people",
+    //   id: 5,
+    //   genre: "Hip Hop",
+    //   userRole: Role.Singer,
+    //   releaseDate: new Date(2018, 5).toString(),
+    //   description: "Description",
+    //   albumImage:
+    //     "https://images.complex.com/complex/images/c_fill,dpr_auto,f_auto,q_90,w_1400/fl_lossy,pg_1/hcjrqlvc6dfhpjxob9nt/cudi",
+    //   contributors: {
+    //     1: { name: "John", role: Role.Producer, stake: 0.25 },
+    //     2: { name: "Dan", role: Role.SoundEngineer, stake: 0.25 },
+    //     3: { name: "Kid Cudi", role: Role.Singer, stake: 0.5 },
+    //   },
+    //   extraInformation: "extra info",
+    // },
+    // 13: {
+    //   name: "Sad people",
+    //   id: 5,
+    //   genre: "Hip Hop",
+    //   userRole: Role.Singer,
+    //   releaseDate: new Date(2018, 5).toString(),
+    //   description: "Description",
+    //   albumImage:
+    //     "https://images.complex.com/complex/images/c_fill,dpr_auto,f_auto,q_90,w_1400/fl_lossy,pg_1/hcjrqlvc6dfhpjxob9nt/cudi",
+    //   contributors: {
+    //     1: { name: "John", role: Role.Producer, stake: 0.25 },
+    //     2: { name: "Dan", role: Role.SoundEngineer, stake: 0.25 },
+    //     3: { name: "Kid Cudi", role: Role.Singer, stake: 0.5 },
+    //   },
+    //   extraInformation: "extra info",
+    // },
+    // 14: {
+    //   name: "Sad people",
+    //   id: 5,
+    //   genre: "Hip Hop",
+    //   userRole: Role.Singer,
+    //   releaseDate: new Date(2018, 5).toString(),
+    //   description: "Description",
+    //   albumImage:
+    //     "https://images.complex.com/complex/images/c_fill,dpr_auto,f_auto,q_90,w_1400/fl_lossy,pg_1/hcjrqlvc6dfhpjxob9nt/cudi",
+    //   contributors: {
+    //     1: { name: "John", role: Role.Producer, stake: 0.25 },
+    //     2: { name: "Dan", role: Role.SoundEngineer, stake: 0.25 },
+    //     3: { name: "Kid Cudi", role: Role.Singer, stake: 0.5 },
+    //   },
+    //   extraInformation: "extra info",
+    // },
```

Comment:
In this case, I think it would be fine to leave the mock data as is, and then just have instructions in the PR description that the reviewer should comment out the code in order to view the changes.